templates/projects/settings.html

```diff
@@ -115,28 +115,32 @@ <h5 class="card-title mb-0">Flavors</h5>
                                 {% csrf_token %}
```

Comment:
Regarding this file, I think that this should be done via admin pannel and not through project settings.

Better add all this additional info to the `help_text` of the model's fields

Reply:
Let's keep this additional form in the Serve UI for future use cases.